<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_NewManufacturer</class>
 <widget class="QWidget" name="Form_NewManufacturer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>204</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>216</width>
    <height>196</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fabricant</string>
  </property>
  <widget class="QPushButton" name="pushButton_AjouterManufacturer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter fabricant</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_NomManufacturer"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Adresse"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Code ZIP</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_ZIP"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Pays</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Country"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajouter un fabricant</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_NomManufacturer</tabstop>
  <tabstop>lineEdit_Adresse</tabstop>
  <tabstop>lineEdit_ZIP</tabstop>
  <tabstop>lineEdit_Country</tabstop>
  <tabstop>pushButton_AjouterManufacturer</tabstop>
  <tabstop>comboBox_Manufacturer</tabstop>
  <tabstop>pushButton_NewManufacturer</tabstop>
  <tabstop>lineEdit_PieceName</tabstop>
  <tabstop>lineEdit_EANPiece</tabstop>
  <tabstop>plainTextEdit_DescriptionPiece</tabstop>
  <tabstop>doubleSpinBox_PrixPiece</tabstop>
  <tabstop>comboBox_Vehicule</tabstop>
  <tabstop>pushButton_NewVehicule</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
